MAINTAINERS

```diff
@@ -63,14 +63,18 @@ maintainer candidates are selected and proposed on the maintainers mailing list.
 
 After a candidate has been announced on the maintainers mailing list, the
 existing maintainers are given five business days to discuss the candidate,
-raise objections and cast their vote. Candidates must be approved by at least 66% of the current maintainers by adding their vote on the mailing
-list. Only maintainers of the repository that the candidate is proposed for are
+raise objections and cast their vote. Candidates must be approved by at least
+66% of the current maintainers by adding their vote on the mailing list. The
+reviewer role has the same process but only requires 33% of current maintainers.
+Only maintainers of the repository that the candidate is proposed for are
```

Comment:
IIUC maintainership is organization-scoped, not repository-scoped?

Reply:
Yes, I think we should probably clarify that as well. This line only got touched for formatting. I would prefer to separate that change unless you want to add a commit to my branch to clarify :smile: